administrator/components/com_content/src/Model/ArticleModel.php

```diff
@@ -784,6 +784,7 @@ public function save($data)
 		if ($input->get('task') == 'save2copy')
```

Comment:
```suggestion
		if ($input->get('task') === 'save2copy')
```
Use type safe comparison if possible,